pkg/deploy/generator/scripts/util-packages.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# Repository and package management related functions
+
+configure_repo_mariner_extended() {
+    local -r extended_repo_config="https://packages.microsoft.com/cbl-mariner/2.0/prod/extended/x86_64/config.repo"
+    curl -sSL "$extended_repo_config" -o /etc/yum.repos.d/mariner-extended.repo
+
+    local -r repo_name="cbl-mariner2.0prodextendedx86_64"
+
+    local -ra cmd=(
+        dnf
+        update
```

Comment:
How much of this do we still need to handle given the automated update functionality?

Reply:
The automated update is handled via an OS image update.  
Meaning, when it's time to update the OS, a new OS image is applied with the latest updates at the time the image was created.  
I imagine it will still have some outdated packages, unless they are created and rolled out on the same day.  

When the VMSS instances are initially created, they will be using an older image.  
This is running to update that older image, I think this is still necessary.  

I have removed our dnf update cronjob, as that's no longer necessary given the automatic updates.